brokers/go/Makefile

```diff
@@ -22,8 +22,8 @@ binary-local:
 	go get -d ./...
 	go build -o $(EXE) -tags netgo -installsuffix netgo gobroker.go
 
-broker:
-	docker run --net-host -ti -p 9090:9090 gobroker
+broker: $(EXE)
+	docker run --net=host -ti -p 9090:9090 gobroker
```

Comment:
yep - you’re right - fixed it

> On Sep 8, 2016, at 1:30 PM, Morgan Bauer <notifications@github.com> wrote:
> 
> In brokers/go/Makefile <https://github.com/servicebroker/servicebroker/pull/78#discussion_r78052261>:
> 
> > @@ -22,8 +22,8 @@ binary-local:
> >  	go get -d ./...
> >  	go build -o $(EXE) -tags netgo -installsuffix netgo gobroker.go
> >  
> > -broker:
> > -	docker run --net-host -ti -p 9090:9090 gobroker
> > +broker: $(EXE)
> > +	docker run --net=host -ti -p 9090:9090 gobroker
> isn't net=host and allocating a specific port exclusive?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/servicebroker/servicebroker/pull/78/files/1d9b803e99636f6591e735f259d56e4d7b3b7253#r78052261>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AB2sX20IcK9tY9m1Um-WkhIiG2pVSPaNks5qoEZTgaJpZM4J4KyM>.
>